### DISCUSSION TITLE
How can I import file or folder from a protected link?

### DISCUSSION THREAD

**Discussion by crskycode** [qgustavor/mega#222] [Q&A]
I want show a password input dialog in my client.

**Comment by qgustavor** [comment]
Someone will need to check how MEGA implemented that and implement it here. I can't even try out this feature as I don't have a Pro plan.

**Edit:** related to https://github.com/qgustavor/mega/discussions/203

**Reply by crskycode** [reply]
> Someone will need to check how MEGA implemented that and implement it here. I can't even try out this feature as I don't have a Pro plan.
> 
> **Edit:** related to #203

okay, got it.
